gum/backend-x86/gumstalker-x86.c

```diff
@@ -407,6 +408,9 @@ static void gum_exec_block_write_unfollow_check_code (GumExecBlock * block,
 static void gum_exec_block_write_call_probe_code (GumExecBlock * block,
     const GumBranchTarget * target, GumGeneratorContext * gc);
 
+static void gum_exec_block_write_sysenter_continuation_code (
+    GumExecBlock * block, GumGeneratorContext * gc, gpointer saved_ret_addr);
```

Comment:
Let's move it right after `gum_exec_block_write_single_step_transfer_code`, without blank lines surrounding it, so the code flows from highest to lowest level.